tests/browser_context_test.go

```diff
@@ -913,3 +913,32 @@ func TestBrowserContextGrantPermissions(t *testing.T) {
 		})
 	}
 }
+
+func TestBrowserContextClearPermissions(t *testing.T) {
+	t.Parallel()
+
+	t.Run("no_permissions_set", func(t *testing.T) {
+		t.Parallel()
+
+		tb := newTestBrowser(t)
+		bCtx, err := tb.NewContext(nil)
+		require.NoError(t, err)
+
+		err = bCtx.ClearPermissions()
+		assert.NoError(t, err)
+	})
+
+	t.Run("permissions_set", func(t *testing.T) {
+		t.Parallel()
+
+		tb := newTestBrowser(t)
+		bCtx, err := tb.NewContext(nil)
+		require.NoError(t, err)
+
+		err = bCtx.GrantPermissions([]string{"geolocation"}, common.NewGrantPermissionsOptions())
+		require.NoError(t, err)
+
+		err = bCtx.ClearPermissions()
+		assert.NoError(t, err)
```

Comment:
Should we also check that permissions are not there?

Reply:
I gave a +1 originally to this, but we currently don't have a method to do this AFAIK, and skimming through the [CDP docs](https://chromedevtools.github.io/devtools-protocol/tot/Page/#type-PermissionsPolicyFeatureState), seems like the action that would allow it is at the page level. So probably this requires a separate PR if desired.

Reply:
Yeah, good spot. I couldn't find a way to assert before and after the permissions were granted. I'm not sure if that's something we need at the moment, and it would depend on whether users request it (and if there's a CDP API to request current applied permissions).